ee/client/omnichannel/cannedResponses/CannedResponseEditWithData.tsx

```diff
@@ -1,15 +1,21 @@
 import { Callout } from '@rocket.chat/fuselage';
-import React, { useMemo } from 'react';
+import React, { FC } from 'react';
 
 import { FormSkeleton } from '../../../../client/components/Skeleton';
 import { useTranslation } from '../../../../client/contexts/TranslationContext';
 import { AsyncStatePhase } from '../../../../client/hooks/useAsyncState';
 import { useEndpointData } from '../../../../client/hooks/useEndpointData';
-import CannedResponseEdit from './CannedResponseEdit';
+import CannedResponseEditWithDepartmentData from './CannedResponseEditWithDepartmentData';
 
-function CannedResponseEditWithData({ cannedResponseId, reload, title }) {
-	const query = useMemo(() => ({ _id: cannedResponseId }), [cannedResponseId]);
-	const { value: data, phase: state, error } = useEndpointData('canned-responses.getOne', query);
+const CannedResponseEditWithData: FC<{
+	cannedResponseId: string;
+	reload: () => void;
+}> = ({ cannedResponseId, reload }) => {
+	const {
+		value: data,
+		phase: state,
+		error,
+	} = useEndpointData(`canned-responses/${cannedResponseId}` as 'canned-responses/${string}');
```

Comment:
As you also did part of my changes on #22736, can you also replicate the changes I did on the other file on that same PR, so I can close mine to avoid conflicts :)